requirements-dev.txt

```diff
@@ -5,4 +5,4 @@ django-debug-toolbar-request-history==0.0.11    # for debugging database queries
 mock                                            # mock objects for unit tests
 responses==0.10.15                               # mock HTTP responses for unit tests
 urllib3-mock==0.3.3                             # mock urllib3 for tests
-Jinja2==2.10.1                                  # jinja templates are used by servctl k8s deployment scripts
+Jinja2==2.11.3                                  # jinja templates are used by servctl k8s deployment scripts
```

Comment:
There was a Snyk error as below:

Vulnerable module: | jinja2
-- | --
Introduced through: | jinja2@2.10.1
Exploit maturity: | Proof of concept
Fixed in: | 2.11.3

Is it okay to make this change?

Reply:
no and you should not be changing python dependencies at all as part of this